src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -609,6 +609,38 @@ Bottom line, the OAuth flow implemented in `step` and `step-ca` is widely vetted
   </div>
 </Alert>
 
+#### Device Authorization Grant Flow
+
+The Device Authorization Grant flow is an OAuth 2.0 extension that enables OAuth for devices with no browser 
+or limited input capability to obtain an access token.
+The OIDC provisioner supports the Device Authorization Grant flow, however,
+it is not always the default flow for a given OIDC provider.
+
+The Google OIDC provider initially defaulted to a different flow - the Out Of Band (OOB) flow which has since been deprecated. 
+`step-ca` OIDC provisioners configured to use Google OIDC will still default to the OOB flow (until October 3rd, 2022).
+Users can specify the proper OAuth 2.0 flow by supplying flags and environment variables.
+
+To use the the default OAuth 2.0 flow for input constrained devices (for your OIDC provider) use:
```

Comment:
The problem is that this isn't currently true. It will always be true after October 3rd. It's currently only true for non-Google providers. We can just say whatever, it will be true soon.